src/gui/main_window.py

```
@@ -0,0 +1,254 @@
+from PyQt6.QtWidgets import (
+    QMainWindow, QTabWidget, QWidget, QVBoxLayout, QLabel,
+    QStatusBar, QToolBar, QMessageBox, QSizePolicy
+)
+from PyQt6.QtCore import Qt, pyqtSignal, QTimer
```

Ruff (0.8.2)

`PyQt6.QtCore.Qt` imported but unused

Remove unused import: `PyQt6.QtCore.Qt`

(F401)